src/Services/catalogapi/pom.xml

```diff
@@ -44,25 +41,23 @@
 			<artifactId>lombok</artifactId>
 			<optional>true</optional>
 		</dependency>
-
 		<dependency>
 			<groupId>io.r2dbc</groupId>
 			<artifactId>r2dbc-mssql</artifactId>
-			<version>1.0.0.RELEASE</version>
-		</dependency>
+			<version>0.9.0.RELEASE</version>
```

Comment:
Changed verson due to this issue
https://github.com/spring-projects/spring-framework/issues/28864